docs/docs/bootstrapping.md

```diff
@@ -40,6 +40,12 @@ Note that if a Maxwell client_id has been set you should specify the client id.
 mysql> insert into maxwell.bootstrap (database_name, table_name, client_id) values ('fooDB', 'barTable', 'custom_maxwell_client_id');
 ```
 
+You can schedule bootstrap tasks in the future, setting the `started_at` column value to the time you want to run. Every time maxwell will start bootstrap tasks with this value is `NULL` or in the past. 
```

Comment:
"You can schedule bootstrap tasks to be run in the future by setting the `started_at` column.  Maxwell will wait until this time to start the bootstrap."